qsr_lib/cfg/defaults.ini

```diff
@@ -1,2 +1,3 @@
 [arg_relations_distance]
-relations_and_values = {"0": 5., "1": 15., "2": 120.}
+;relations_and_values = {"0": 5., "1": 15., "2": 120.}
+relations_and_values = {}
```

Comment:
Is this intentional or just to test the new feature? Since the config file is optional I don't think having an empty dict in there is helpful or necessary.